cmd/kots/cli/reset-password.go

```diff
@@ -56,6 +56,11 @@ func ResetPasswordCmd() *cobra.Command {
 				return errors.Wrap(err, "failed to set new password")
 			}
 
+			// delete all sessions to force password change
+			if err := deleteAllSessions(namespace); err != nil {
```

Comment:
From a security perspective (I also lack full context, so feel free to point that out):

What happens if `deleteAllSessions` errors here?
- The user gets a exit code `0`.
- The user gets a log statement saying it didn't delete all sessions (I can guarantee someone isn't going to read it if they get `0`).
- A compromised session may still exist.

Other possibly alternatives (in no particular order);
- Error here
  - Compromised sessions may still exist
  - Rerunning the command may produce the same error (though hopefully it's a transient error)
- Put in some kind of retry logic (definitely a route I'd avoid unless we have to/have a pattern for it)
  - This way transient errors are hopefully caught
- Some kind of db style transaction and roll back the password
  - Feels very ugly, and hard to implement if there's no precedent for it
- Make a manual delete sessions CLI command to ensure if this step fails, the user can try just the last bit (and error out here to highlight they have work to do)
  - Adds an edge case for users, but allows them to have more granular control over their password management.
 
In summary: Maybe line 61 needs to be an error to bring attention to the user that they _may_ still have something to do

Reply:
@trelore I have added a session check, if a session was created before the passwordUpdatedAt, we invalidate session,  try deleting again and log the user out.
it should address the above comment
https://github.com/replicatedhq/kots/pull/2739#discussion_r856450037